Sprint-2/stretch/till.js

```diff
@@ -21,11 +32,16 @@ const till = {
   "20p": 10,
 };
 const totalAmount = totalTill(till);
+console.log(totalAmount);
 
+module.exports = totalTill;
 // a) What is the target output when totalTill is called with the till object
+// £NaN
```

Comment:
The _target_ output is what we _expect_, not what we actually see - what do we actually expect the answer to be?